### PR TITLE
moved resource loading locations and demo page now changes

### PR DESCRIPTION
**Pull request by Georgehe4**

### CODE REVIEW

File: web/embeddings_demo.html [hunk 1]
```diff
@@ -53,8 +53,20 @@
             velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat
             cupidatat non proident, sunt in culpa qui officia deserunt mollit anim
             id est laborum.
+            <select id ="select" onChange = "changeOptions()">
```
Comment (devoncarew) [inline web/embeddings_demo.html]: nit: no spaces around the `=`

File: web/embeddings_demo.html [hunk 2]
```diff
@@ -87,5 +99,11 @@
       <div class="col-md-2"></div>
     </div>
   </body>
-
+  <script>
+  function changeOptions(){
```
Comment (devoncarew) [inline web/embeddings_demo.html]: nit: indent the JS code w/ 2 spaces

File: web/embeddings_demo.html [hunk 3]
```diff
@@ -87,5 +99,11 @@
...
+    var select = document.getElementById("select");
+    var url = select.options[select.selectedIndex].value + '/embed-dart.html';
```
Comment (devoncarew) [inline web/embeddings_demo.html]: let's reverse this: `/embed-dart.html?gist=23hj4gjh2g34`

**Comment by devoncarew** [comment]
Is this PR still relevant?

**Comment by Georgehe4** [comment]
No. Closing

**Pull request closed without merging**